x/staking/types/commission.go

```diff
@@ -10,18 +10,18 @@ import (
 type (
 	// Commission defines a commission parameters for a given validator.
 	Commission struct {
-		Rate          sdk.Dec   `json:"rate"`            // the commission rate charged to delegators
-		MaxRate       sdk.Dec   `json:"max_rate"`        // maximum commission rate which this validator can ever charge
-		MaxChangeRate sdk.Dec   `json:"max_change_rate"` // maximum daily increase of the validator commission
+		Rate          sdk.Dec   `json:"rate"`            // the commission rate charged to delegators, as a fraction
+		MaxRate       sdk.Dec   `json:"max_rate"`        // maximum commission rate which this validator can ever charge, as a fraction
+		MaxChangeRate sdk.Dec   `json:"max_change_rate"` // maximum daily increase of the validator commission, as a fraction
 		UpdateTime    time.Time `json:"update_time"`     // the last time the commission rate was changed
 	}
 
 	// CommissionMsg defines a commission message to be used for creating a
 	// validator.
 	CommissionMsg struct {
-		Rate          sdk.Dec `json:"rate"`            // the commission rate charged to delegators
-		MaxRate       sdk.Dec `json:"max_rate"`        // maximum commission rate which validator can ever charge
-		MaxChangeRate sdk.Dec `json:"max_change_rate"` // maximum daily increase of the validator commission
+		Rate          sdk.Dec `json:"rate"`            // the commission rate charged to delegators, as a fraction
+		MaxRate       sdk.Dec `json:"max_rate"`        // maximum commission rate which validator can ever charge, as a fraction
+		MaxChangeRate sdk.Dec `json:"max_change_rate"` // maximum daily increase of the validator commission, as a fraction
```

Comment:
All the above comment changes are actually technically incorrect, these are decimal types not fractions

Reply:
hmm, I mean that they're fractions as opposed to percentages

Reply:
what would you write instead?

Reply:
Oh I see the intention - good enough